platform/macos/src/Mapbox.h

```diff
@@ -1,10 +1,12 @@
 #import <Cocoa/Cocoa.h>
 
+#import "MGLFoundation.h"
+
 /// Project version number for Mapbox.
-FOUNDATION_EXPORT double MapboxVersionNumber;
+FOUNDATION_EXPORT MGL_EXPORT double MapboxVersionNumber;
```

Comment:
From a C build perspective, this is very weird. `FOUNDATION_EXPORT` is only macro for `extern`, meaning it should be defined in some translation unit (but it's not). Should we manually define this version number?